grafana-plugin/integration-tests/users/usersActions.test.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect, Page } from '../fixtures';
+import { goToOnCallPage } from '../utils/navigation';
+
+test.describe('view list of users', () => {
+  test.slow(); // this test is doing a good amount of work, give it time
+
+  test('admin is allowed to edit other profiles', async ({ adminRolePage }) => {
+    const { page } = adminRolePage;
+
+    await goToOnCallPage(page, 'users');
+
+    const usersTableElement = page.getByTestId('users-table');
+    await usersTableElement.waitFor({ state: 'visible' });
+
+    const editOtherUserProfileDisabledButtonList = await page.locator('button.edit-other-profile-button[disabled]');
+
+    expect(editOtherUserProfileDisabledButtonList).toHaveCount(0);
```

Comment:
I would deduplicate all of this into a function just above, and the only argument to the function would be the `Locator` object that differ between the admin/editor tests, this way you can reuse this logic